services/brig/src/Brig/Email.hs

```diff
@@ -49,7 +49,7 @@ import Imports
 import Network.Mail.Mime
 
 -------------------------------------------------------------------------------
-sendMail :: Mail -> (AppIO r) ()
+sendMail :: (MonadIO f, MonadReader Env f) => Mail -> f ()
```

Comment:
```suggestion
sendMail :: (MonadIO m, MonadReader Env m) => Mail -> m ()
```

Reply:
Sure, I applied this suggestion. While your suggestion works because values and types live in different name spaces in Haskell, I tend to avoid name ambiguity as I find it easier to read code when one name can be only one thing in the given context.